src/main/components/DataTree.jsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import ReactJson from 'react-json-view';
+
```

Comment:
What is this file used for? When using the app, I did not see anything resembling a tree (unless it's either more abstract or I just missed a feature).